management/db.py

```diff
@@ -448,10 +448,39 @@ def delete_freezer(user_id,victim_id):
     conn.commit()
     return True
 
-def add_standoff():
-    """Add a new kill condition to the database."""
-    pass
-    # TODO
+def add_standoff(victim_id,role,murderer):
+    # NOT TESTED YET - PLEASE DO
+    """Add a new order to the kill queue.
+
+    Keyword arguments:
+    victim_id -> the id of the victim to be attacked
+    role -> the role of the attacker
+    murderer -> id of the attacker (random attacker if multiple)
+    """
+    data = [victim_id,role,murderer]
+    c.execute("INSERT INTO 'standoff' ('id','victim','role','murderer') VALUES (NULL,?,?,?)",data)
```

Comment:
Is the `NULL` on `id` here intentional?

Reply:
will setting `NULL` overwrite the autoincrement?

Reply:
In contrary, the autoincrement will overwrite the `NULL`. I used `NULL` as a placeholder, or, I should say, the SQL database manager told me to.